### PR TITLE
Added preserving filters feature

### PR DESCRIPTION
**Pull request by jtkpiotr**
@adamzima @pbzymek @s1mplex @mhib @quirk0o @lkudyba

### CODE REVIEW

File: client/test/helpers/FiltersHelper.spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { expect } from 'chai';
+import { Set } from 'immutable';
+import * as FiltersHelper from 'helpers/FiltersHelper';
+import * as LocalStorageHelper from 'helpers/LocalStorageHelper';
+import sinon from 'sinon';
+import proxyquire from 'proxyquire';
+
+describe('FiltersHelper', () => {
+  before(() => {
+    sinon.stub(LocalStorageHelper, 'setItem');
```
Comment (adamzima) [inline client/test/helpers/FiltersHelper.spec.js:10]: Unstub me like there's no tomorrow!

Comment (jtkpiotr) [inline client/test/helpers/FiltersHelper.spec.js:10]: LocalStorageHelper.setItem is unstubbed in after() hook

File: client/app/helpers/LocalStorageHelper.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export function getItem(key) {
```
Comment (madejejej) [inline client/app/helpers/LocalStorageHelper.js:1]: Are you sure we need this helper?

Comment (jtkpiotr) [inline client/app/helpers/LocalStorageHelper.js:1]: I've made it to be able to easily mock localStorage. It's just a simple wrapper

**Pull request merged**